tests/tests_e3/anod/test_sandbox.py

```diff
@@ -243,3 +244,53 @@ def test_anod_plan(git_specs_dir):
                'test e3 for %s' % platform]
     for action in actions:
         assert action in p.out
+
+
+def test_anodtest(git_specs_dir):
+    """Test the procedure of anodtest('e3')."""
```

Comment:
use something simpler for our testsuite, we don't want to clone a remote repository